config/initializers/devise.rb

```diff
@@ -247,9 +247,9 @@
     }
 
     config.omniauth :github,
-      Errbit::Config.github_client_id,
-      Errbit::Config.github_secret,
-      github_options
+                    Errbit::Config.github_client_id,
+                    Errbit::Config.github_secret,
+                    github_options
```

Comment:
The style-switch in 52d8b62cbc0b6733bf5c00bad939d3e2be9c352d also fixes this case, thankfully.